src/pudl/analysis/record_linkage/embed_dataframe.py

```diff
@@ -69,37 +73,36 @@ def as_pipeline(self):
         )
 
 
-def dataframe_embedder_factory(vectorizers: dict[str, ColumnVectorizer]):
```

Comment:
I'm pretty confused why I was unable to use this `dataframe_embedder_factory`. Basically, when I created another instance of the factory in the FERC to EIA model, it created duplicate nodes with the same name as the ops in the FERC to FERC model. When I moved the two ops and graph outside of this factory function then I was able to call the graph twice, once in each of the two different models. Maybe it should somehow be made into a class? It would be nice to bundle the two ops and graph into one container.